src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1778,6 +1781,25 @@ internal FieldInfo GetField(RuntimeFieldHandleInternal field)
         #region Static Members
 
         #region Internal
+        internal static unsafe RuntimeType? GetParentType(RuntimeType type)
+        {
+            TypeHandle typeHandle = type.GetNativeTypeHandle();
```

Comment:
```suggestion
        internal unsafe RuntimeType? GetParentType()
        {
            TypeHandle typeHandle = GetNativeTypeHandle();
```
This can be instance method.

Reply:
Yes, I agree. I didn't make it an instance method but the type also has a `GetBaseType()` and I thought that was confusing.

Reply:
Is there a win of some sort if the method is an instance vs static?